dbfit-java/core/src/main/java/dbfit/api/DbObject.java

```diff
@@ -1,12 +1,11 @@
 package dbfit.api;
 
-import dbfit.fixture.StatementExecution;
 import dbfit.util.DbParameterAccessor;
 import dbfit.util.Direction;
 
 import java.sql.SQLException;
 
 public interface DbObject {
-    public StatementExecution buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException ;
+    public DbCommand buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException;
```

Comment:
> Should we align this with the returned type? E.g. `buildDbCommand`.

Not necessarily aligned with the type name but should be aligned with what it does and `buildDbCommand` sounds to me a better fit for that purpose too. So I renamed it.